CONTRIBUTING.md

```diff
@@ -147,6 +147,13 @@ follow the instructions [below](#integrating-the-regression-test-with-cmake) *an
 `perf-reg` label to the test (see bottom of the regression
 [CMakeLists.txt](tst/regression/CMakeLists.txt)).
 
+A third pipeline is run using LANL internal systems and is run manually when
+approved, and is scheduled to run on a dailly basis on the development branch.
+These machines use the newest IBM powerPC processors and the NVIDIA V100
+(Volta) GPUs. Tests run on these systems are primarily aimed at measuring the
+performance of this specific architecture. Compilation and testing details can be found by looking
+in the [.gitlab-ci-darwin.yml](.gitlab-ci-darwin.yml) file *and* the /scripts/darwin folder.. 
```

Comment:
I think it'd be great to be a little more explicit here.
For example, I'm currently not sure what we're actually testing in terms of architecture and tests.
Based on the current description I would expect 
- unit performance test using a single core and single host thread, or a single GPU,
- the performance regression test using CPUs  (serial and with some MPI processes) with one thread per process
- the performance regression test using a single process with a single GPUs, and multiple processes with multiple GPUs

I think (without knowing what's in the logs) this may not match what's currently tested,

Reply:
This is a fair point.